x/yarpctest/request_unary.go

```diff
@@ -207,3 +266,12 @@ func GiveAndWantLargeBodyIsEchoed(numOfBytes int) api.RequestOption {
 		opts.WantResponse.Body = ioutil.NopCloser(bytes.NewReader(body))
 	})
 }
+
+// WithRetry retries the request for a given times, until the request succeeds
+// and the response matches.
+func WithRetry(count int, interval time.Duration) api.RequestOption {
```

Comment:
Drop `With` since we don't prefix it anywhere else in this test suite

Reply:
done